conda-store-server/conda_store_server/schema.py

```diff
@@ -34,6 +34,7 @@ class Permissions(enum.Enum):
     BUILD_DELETE = "build::delete"
     NAMESPACE_CREATE = "namespace::create"
     NAMESPACE_READ = "namespace::read"
+    NAMESPACE_UPDATE = "namespace::update"
```

Comment:
@pierrotsmnrd this PR looks great to me. No real issues. My questions is on the granularity of permissions:

Should we have one permission `namespace::update` or should we have two? To me having the ability to change a namespaces metadata and giving users permissions to a namespace are at much different permission levels.

So for example:
  - `namespace::update`
  - `namespace-role-mapping::create`
  - `namespace-role-mapping::delete`

I'm not entirely sure on the naming.